### ISSUE TITLE
Reprojection Filtering

### ISSUE THREAD

**Issue by jonmatthis** [freemocap/freemocap#417]
give users more control over the charuco board definitions and easier feedback to let them know if they are doing thigns properly

**Comment by jonmatthis** [comment]
blocked by #412

**Comment by trentwirth** [comment]
@philipqueen #466 - PR on new reprojection error filtering method

**Comment by trentwirth** [comment]
Jon will check out the blender file Philip has on the NAS - go on from there

**Comment by trentwirth** [comment]
Philip and Aaron are testing this now to make sure it's ready to go 🚀

**Comment by trentwirth** [comment]
A bunch of changes were required because it touches on the `process recording folder` 

Philip is waiting to merge it into development until someone (in addition to Philip having already done it) completes a multi-camera recording w/ the reprojection error addition

**Comment by trentwirth** [comment]
This runs, doesn't break anything - have some small evidence that it's improving things. BUT, to know it's doing *work*, it will/would require more investigation. This will get picked up at a later date ✨
